docs/manual/key-storage/storage-plugins/thycotic-storage.md

```diff
@@ -12,21 +12,28 @@ Process Automation users have access to the Thycotic Storage Plugin which can be
 
 Below is an example configuration, which can be configured using the *System Configuration* module. Add each setting as a configuration entry.
 
-![Thycotic Configuration](@assets/img/thycotic-config-screen.png)
+:::warning
+Doing it via Configuration Manager Plugin (GUI way to write configuration) will expose sensible fields such as the password. We recommend to use "$RDECK_BASE/rundeck-config.properties" file.
+:::
+
+![Thycotic Configuration](@assets/img/thycoticConfiguration.png)
+
+**Note:** if user use Thycotic Storage Plugin as a second storage provider, and its path preffix is set to ```true```, the keys will be downloaded to a separate folder within "keys" folder, exhibit in the following:
```

Comment:
This doesn't need a special "Note".  It's a common setting across all the Key Storage plugins.

Reply:
ok, i'll remove it